scripts/sct_image.py

```diff
@@ -204,6 +211,13 @@ def main(args=None):
         im_out = [im_dest_new]
         fname_out = arguments.copy_header
 
+    elif arguments.set_sform_to_qform:
```

Comment:
should it be an elif or an isolated if? ie: the copy of the sform could potentially be done _in addition to_ other processes, couldn't it?

Reply:
Definitely. So the idea would be to set the sform to the qform for the input image before other manipulations?

Reply:
hum... i would say so. I don't think it matters given that the current functions in sct_image only use the qform, but it is probably wiser to do it at the beginning. And hopefully there is no process that alters _only_ the qform 😬

Reply:
This refactor is slightly more involved than I thought. The input image is read in multiple times throughout the the script, so unless we do an `elif` then we're going to need to alter the code in multiple places.

Reply:
I thought of a reasonably simple fix, just replace `Image` with the sform fixer code if the user wants.